tests/protos/kitchen-sink.proto

```diff
@@ -7,6 +7,7 @@
 package some_package;
 
 import "tests/protos/options.proto";
+import "google/protobuf/descriptor.proto";
```

Comment:
Oh. It looks like options.proto imports it, or does it not actually import it? 

I just meant if I were to extend another message in the same file, I'd have to add support for "extensions 10 to 20;" syntax to specify that those numbers are available for extending, otherwise another test complains.